app/assets/stylesheets/login_modal.scss

```diff
@@ -8,17 +8,21 @@
   padding: 0 30px;
 
   a#visibility {
-    line-height: 1.5em;
-    color: #888;
-    pointer-events: auto;
-    text-decoration: none;
+    line-height: 0;
+  }
+
+  div.form-group {
+    margin-top: 1rem;
   }
-}
 
-div.form-group {
-  margin-top: 1rem;
+  p.form-grey {
+    margin-top: 1rem;
+  }
 }
 
-p.form-grey {
-  margin-top: 1rem;
+#visibility {
```

Comment:
I've just realized now that you can also put visibility in .login-form(like you did originally) and replace 2.2em with 0:

```css
.login-form {
    #visibility{
        color: #888;
        pointer-events: auto;
        line-height: 0;
        text-decoration: none;
    }
}
```